src/lightning/app/cli/connect/app.py

```diff
@@ -77,7 +77,9 @@ def connect_app(app_name_or_id: str):
             if app_name_or_id == "localhost":
                 click.echo("You are connected to the local Lightning App.")
             else:
-                click.echo(f"You are already connected to the cloud Lightning App: {app_name_or_id}.")
+                click.echo(
+                    f"You are already connected to the cloud Lightning App: {app_name_or_id}. If it isn't running. Either restart it or reconnect to access lightning main cli."
```

Comment:
```suggestion
                    f"You are already connected to the cloud Lightning App: {app_name_or_id}. If it isn't running, either restart it or reconnect to access the lightning cli."
```

Reply:
Note, the line is too long and the formatter complains. We should break it into two lines :)